bin/dustc

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env node
 
 var cli = require('cli').enable('glob', 'version'),
+    watchr = require('watchr'),
```

Comment:
OK, well, it doesn't require it-- it runs without it if it's not available. I left both commits in so that we can easily revert a single commit if it turns out to be problematic.